runner/src/main/java/com/mulesoft/tools/migration/MigrationRunner.java

```diff
@@ -46,12 +47,14 @@
   private final static String REPORT_HOME = "summary.html";
   public static final String MULE_3_VERSION = "3.*.*";
   private final static String CANCEL_ON_ERROR = "cancelOnError";
+  private final static String PROJECT_PARENT_GAV = "projectParentGAV";
 
   private String projectBasePath;
   private String parentDomainProjectBasePath;
   private String destinationProjectBasePath;
   private String muleVersion;
   private boolean cancelOnError = false;
+  private String projectParentGAV;
```

Comment:
Just, a little suggestion: 
Why not use com.mulesoft.tools.migration.project.model.pom.Parent class? 
Using this data structure there is no need to split the GAV argument in the UpdateProjectParent Step. You can create this Parent instance after validating the parameter.